src/main/java/com/mixpanel/android/mpmetrics/MixpanelFCMMessagingService.java

```diff
@@ -140,7 +140,11 @@ public static void showPushNotification(Context context, Intent messageIntent) {
         Notification notification = mixpanelPushNotification.createNotification(messageIntent);
         if (null != notification) {
             final NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
-            notificationManager.notify(0, notification);
+            if (mixpanelPushNotification.tag != null) {
+                notificationManager.notify(mixpanelPushNotification.tag ,0, notification);
```

Comment:
remove space after `.tag` and add space after the comma: `notificationManager.notify(mixpanelPushNotification.tag, 0, notification);`